src/visualization/types.jl

```diff
@@ -557,9 +557,11 @@ function PlotData1D(u, mesh, equations, solver, cache;
   if ndims(mesh) == 1
     x, data, mesh_vertices_x = get_data_1d(original_nodes, unstructured_data, nvisnodes)
     orientation_x = 1
-  else # ndims(mesh) == 2
+  elseif ndims(mesh) == 2
     pd = PlotData2DTriangulated(u, mesh, equations, solver, cache; solution_variables, nvisnodes)
-    x, data, mesh_vertices_x = unstructured_2d_to_1d_curve(pd, curve, slice, point)
+    x, data, mesh_vertices_x = unstructured_2d_to_1d_curve(pd, curve, slice, point, nvisnodes)
+  else # ndims(mesh) == 3
+    x, data, mesh_vertices_x = unstructured_3d_to_1d_curve(original_nodes, u, curve, slice, point, nvisnodes)
```

Comment:
Out of curiosity: Why do you need to pass different arguments in 2D and 3D (`original_nodes, u` instead of `pd`)?

Reply:
Please answer this question

Reply:
@FelipeSantillan?

Reply:
My answer is below.

Reply:
Thanks for clarifying. Can you add this clarification as a comment above the call to `unstructured_3d_to_1d_curve` here?